app/assets/stylesheets/modules/toolbar.css.scss

```diff
@@ -15,8 +15,31 @@
   margin: 0;
 }
 
+.manual-coord-entry {
```

Comment:
I don't so much like the number of one-off styles required for this feature. Could some of it be reusable?